spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/Info.java

```diff
@@ -84,4 +84,21 @@ public String getDescription() {
 	public void setDescription(String description) {
 		this.description = description;
 	}
+
+	/**
+	 * Create a new Info instance with the given description, first deployed and last deployed dates
+	 * set to the current date-time, and StatusCode.UNKNOWN.
+	 * @param description a string describing the general info status at a level finer than StatusCode
+	 * @return a new Info object
+	 */
+	public static Info createNewInfo(String description) {
```

Comment:
just comment for future as we probably want to overhaul other pojos as well for static methods and think if we want to add Builder interfaces. Naming will easily get off the rails.

Reply:
Please create an issue to add the builder pattern here....

Reply:
So far, I think this is the only case, but we do have a few other sections with this type of 'boilerplate' code which could be moved down into the domain objects....